### PR TITLE
Pull badges from the DB and calculate for each user on the backend

### PR DESCRIPTION
**Pull request by alyssadelaine**
Pulling from the database is still in progress

### CODE REVIEW

**Comment by alyssadelaine** [comment]
Actually @kamicut I think I should do some more things before we merge this

File: frontend/src/components/BadgeBox.js [hunk 1]
```diff
@@ -51,12 +27,14 @@ function stripWS(text) {
 export default ({ badges }) => {
   if (badges && badges.all) {
     var progressBadges = Object.keys(badges.all).map(val => {
+      console.log(badges)
```
Comment (kamicut) [inline frontend/src/components/BadgeBox.js]: remove these console.log

File: api/src/badge_logic/badges.js [hunk 2]
```diff
@@ -1,51 +1,52 @@
-export default {
+module.exports = {
```
Comment (kamicut) [inline api/src/badge_logic/badges.js]: Why do we still need this file?

Comment (alyssadelaine) [reply api/src/badge_logic/badges.js]: You're right, we don't. I'll remove this and the console.log @kamicut

**Pull request merged**